CMakeLists.txt

```diff
@@ -193,6 +193,7 @@ option(LEXER_DEBUG "Build lexer with debug option" OFF)
 option(USE_GIT_COMMIT "Show the git commit in version information" ON)
 option(REQUIRE_PTHREADS "For non-Windows systems, set whether Csound will use threads or not" ON)
 option(BUILD_DOCS "Build documentation (requires doxygen)" OFF)
+option(DEBUG_ERROR_ON_WARNING "For a debug build, error on a compiler warning" ON)
```

Comment:
This should default to off. Until we make an effort to actually reduce the warnings. Otherwise there will be build breakages where they weren't failing before.

Reply:
I agree, but `-Werror` is on by default and I didn't want to change things.

Reply:
Ok, turned it off by default